daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Engine.scala

```diff
@@ -145,11 +146,12 @@ final class Engine {
     * If let undefined, no discriminator will be generated.
     */
   def reinterpret(
```

Comment:
Please update the comment according to the changed parameters:
- there is no transactionSeed any more
- the seed argument and the submission time should be added to `reinterpret(cmds.submitter, ...)` and `reinterpret(n.requiredAuthorizers, ...)`
- renaming of relative contract IDs is now suffixing of contract IDs

I also recommend to extend the comment with a warning that it is not sensible to run `reinterpret` for a several nodes unless they are not exactly the roots of the original transaction.